test/acceptance/tests.py

```diff
@@ -875,6 +913,8 @@ def test_staff_override_at_beginning(self):
         And I see my staff override score
         And all fields in the staff area tool are correct
         """
+        # Note: this test skips self.do_train_self_peer(), as it needs more fine-grained validation and control.
```

Comment:
I don't understand this comment-- I intentionally don't want to do the training, self, and peer steps at the beginning of the test.